dnf-docker-test/features/installroot-metadata-install.feature

```diff
@@ -0,0 +1,153 @@
+ When I execute "dnf" command "dnf --installroot=/dockertesting config-manager --add-repo http://127.0.0.1/repo/test-1-gpg/test-1-gpg-file.repo" with "success"
+ When I execute "dnf" command "dnf --installroot=/dockertesting repolist test-1-g*" with "success"
+ Then line from "stdout" should "start" with "test-1-gpg-file"
+
+Scenario: GPG key import after package install plus dnf.log test in installroot
+ When I execute "bash" command "rpm --root=/dockertesting -q gpg-pubkey --qf '%{name}-%{version}-%{release} --> %{summary}\n'" with "fail"
+ When I execute "bash" command "rpm -q gpg-pubkey --qf '%{name}-%{version}-%{release} --> %{summary}\n'" with "success"
+ Then line from "stdout" should "not start" with "gpg-pubkey-2d2e7ca3-56c1e69d --> gpg(DNF Test1 (TESTER) <dnf@testteam.org>)"
+# It also test if dnf.log file is used from installroot if package installed in installroot
+ When I execute "bash" command "rm -f /var/log/dnf.log" with "success"
+ Then the path "/var/log/dnf.log" should be "absent"
+ When I execute "bash" command "rm -f /dockertesting/var/log/dnf.log" with "success"
+ Then the path "/dockertesting/var/log/dnf.log" should be "absent"
+ When I execute "dnf" command "dnf --installroot=/dockertesting -y --disablerepo=* --enablerepo=test-1-gpg-file install TestA" with "success"
+ Then the path "/dockertesting/var/log/dnf.log" should be "present"
+ When I execute "bash" command "rpm -q --root=/dockertesting TestA TestB" with "success"
+ Then line from "stdout" should "start" with "TestA-1.0.0-1."
+ And line from "stdout" should "start" with "TestB-1.0.0-1."
+ When I execute "bash" command "rpm --root=/dockertesting -q gpg-pubkey --qf '%{name}-%{version}-%{release} --> %{summary}\n'" with "success"
+ Then line from "stdout" should "start" with "gpg-pubkey-2d2e7ca3-56c1e69d --> gpg(DNF Test1 (TESTER) <dnf@testteam.org>)"
+ When I execute "bash" command "rpm -q gpg-pubkey --qf '%{name}-%{version}-%{release} --> %{summary}\n'" with "success"
+ Then line from "stdout" should "not start" with "gpg-pubkey-2d2e7ca3-56c1e69d --> gpg(DNF Test1 (TESTER) <dnf@testteam.org>)"
+
+Scenario: Install signed package with signed dependece with key from host from repository with gpgcheck=1
+ When I execute "dnf" command "dnf --installroot=/dockertesting -y --disablerepo=* --enablerepo=test-1-gpg-file install TestF" with "fail"
+ When I execute "bash" command "rpm -q --root=/dockertesting TestF TestG TestH" with "fail"
+ Then line from "stdout" should "not start" with "TestF"
+ And line from "stdout" should "not start" with "TestG"
+ And line from "stdout" should "not start" with "TestH"
+
+Scenario: Remove package from installroot
+  When I execute "dnf" command "dnf remove --installroot=/dockertesting --releasever=23 -y TestC" with "success"
+  When I execute "bash" command "rpm -q --root=/dockertesting TestC" with "fail"
+
+Scenario: Tracking information are stored in host persist-dir if package installed in host
```

Comment:
Tracking information *is* stored in host persist-dir if package *is* installed in host